### ISSUE TITLE
Isocurve

### ISSUE THREAD

**Issue by marco-mf3** [vispy/vispy#1414]
When I run isocurve.py
on Windows 7, Python 3.6.2, Vispy 0.5.2, 

`"C:\....\python.exe" C:/emacs/wx_vispy_01/examples/basics/scene/isocurve.py
**WARNING: Unable to disable rotation.**
Traceback (most recent call last):
  File "C:/..../examples/basics/scene/isocurve.py", line 41, in <module>
    parent=view.scene)
  File "C:\....\lib\site-packages\vispy\scene\visuals.py", line 126, in __init__
    subclass.__init__(self, *args, **kwargs)
  File "C:\....\lib\site-packages\vispy\visuals\isocurve.py", line 66, in __init__
    self.set_data(data)
  File "C:\....\lib\site-packages\vispy\visuals\isocurve.py", line 110, in set_data
    self._iso = cntr.Cntr(self._X, self._Y, self._data.astype(float))
**ValueError: Arguments x, y, z, mask (if present) must have the same dimensions.**

Process finished with exit code 1`

If you Google search the value error seems that same issue on Matplotlib and Chaco.

**Comment by kmuehlbauer** [comment]
@marco-mf3 Thanks for the report. This was fixed recently with #1404. You would need to use latest vispy from github. If you are using masked arrays have a look at #1405.

Please report back, if this fixes your issue.

**Comment by marco-mf3** [comment]
Thanks.
with 0.6.0 dev0 is PERFECT (and beautiful... and fast)

**Comment by kmuehlbauer** [comment]
@marco-mf3 Great, that it works for you! And beautiful... and fast, very good!

**Comment by adichell** [comment]
Hi, 
I have the same issue with vispy 0.6.6.

I have a windows 7 64-bit, and I used python 3.7.9, and I have tried to run the isocurve.py example.

I get the same warning " WARNING: Unable to disable rotation. " and then the application freezes and stops running, but I don't get other messages, just this warning.

I replied here since the warning and the behavior seems quite similar, but if you prefer I could open a new issue.
